src/main/java/ch/njol/skript/conditions/CondStartsWith.java

```diff
@@ -50,11 +50,11 @@
 	private Expression<String> strings;
 	@SuppressWarnings("null")
 	private Expression<String> prefix;
-	private boolean ends;
+	boolean ends;
```

Comment:
what the hell are you changing all these to package-private

Reply:
I stated above

Reply:
In simple words, it is to avoid warnings about synthetic-access since I am also accessing that variable in an inner class.

Reply:
^